internal/view/live_view_test.go

```diff
@@ -39,14 +36,13 @@ func TestLiveView_linesWithRegions(t *testing.T) {
 			e: []string{"foo", `<<<"search_0">>>bar<<<"">>>`, `<<<"search_1">>>baz<<<"">>>`},
```

Comment:
In the test you could use a formatter helpers ie matchTag(i int, s string) string. Easier to track should syntax changes or for new test cases.